packages/body/tests/site/test.scss

```diff
@@ -0,0 +1,4 @@
+@import '../../../core/dist/sass/module.scss';
+@import 'module.scss';
+@import '../../../../.templates/test-assets/style.scss';
+@include versioning-check();
```

Comment:
The versioning check has to happen at the very end after all imports. https://github.com/dominikwilkowski/sass-versioning